docs/iris/example_code/General/coriolis_plot.py

```diff
@@ -0,0 +1,99 @@
+"""
+Coriolis parameters over the UK
+===============================
+
+This example shows the coriolis parameters for the area encompassing the UK.
+
+It creates and plots a map showing the coriolis parameters, overlaid on top of
+the UK coastline. The code also creates a custom cube to store the data in and
+to plot it from.
+
+"""
+
+import cartopy.crs as ccrs
+
+import iris
+
```

Comment:
I disagree. In the context of an example program iris should be considered as a third party library just like cartopy or matplotlib etc.

If you were writing a program yourself that used iris you would consider it a third party library so there is no reason we shouldn't do that here in our own examples (which are not part of the iris package).